config/webpack/environment.js

```diff
@@ -14,7 +14,7 @@ const sassLoader = environment.loaders.get('sass');
 // Note: This option is renamed `additionalData` in newer versions of `sass-loader`.
 // Note: Import paths for USWDS required imports have improved in newer versions.
 //       See: https://github.com/uswds/uswds/blob/50f6ffd6/src/stylesheets/packages/_required.scss
-sassLoader.use.find(({ loader }) => loader === 'sass-loader').options.data = `
+sassLoader.use.find(({ loader }) => loader === 'sass-loader').options.prependData = `
```

Comment:
In case you're wondering, the option has been renamed twice, and is still out of date as of the latest version of Webpacker 😅 

`data` -> `prependData` -> `additionalData`